### PR TITLE
Breakout geofence breach status checks from Commander:run(), apply Google Style to Commander Private methods

### PR DESCRIPTION
**Pull request by mcsauder**
## Describe problem solved by this pull request
This PR breaks out geofence checks from the Commander::run() method to improve code maintainability.  This PR also extends the `geofence_result` message to provide for identification of altitude and max distance violations, as well as modifying naming within the message to lay groundwork for a follow-on secondary geofence.

This PR also applies Google Style naming to the Commander class private methods and organizes variables and method ordering in `Commander.hpp`.  Additional renaming in the Commander Public methods was avoided in this PR to minimize the diff in this PR.

## Describe your solution
Continuing to break out focused sections of Commander::run() so that in the end it is essentially a sequential series of very focused method calls is good practice and will drive the code toward a state that eventually can have better unit test coverage applied.  This PR moves the needle further in that direction. 

## Describe possible alternatives
The current code functions and could be left as-is, however that is clearly not the intention of submitting this PR.

## Test data / coverage
Tested in SITL for inclusion boundaries with and without geofence avoidance enabled.  All previous functionality and unit test behavior has been preserved.

## Additional context
This PR simplifies the work being done in #20055

Please let me know if you have questions on this PR.

Thanks!

-Mark

### CODE REVIEW

File: src/modules/commander/Commander.cpp [hunk 1]
```diff
@@ -1862,13 +1859,13 @@ Commander::run()
 
 		handleAutoDisarm();
 
-		if (_geofence_warning_action_on
+		if ((_primary_geofence_warning_action_on)
```
Comment (bkueng) [inline src/modules/commander/Commander.cpp:1862]:
```suggestion
		if (_primary_geofence_warning_action_on
```

File: src/modules/commander/Commander.cpp [hunk 2]
```diff
@@ -1883,120 +1880,12 @@ Commander::run()
...
-				case (geofence_result_s::GF_ACTION_LOITER) : {
-						if (TRANSITION_CHANGED == main_state_transition(_vehicle_status, commander_state_s::MAIN_STATE_AUTO_LOITER,
-								_vehicle_status_flags,
-								_commander_state)) {
-							_geofence_loiter_on = true;
-						}
-
-						break;
-					}
+		checkGeofenceStatus();
```
Comment (bkueng) [inline src/modules/commander/Commander.cpp:1883]: Just fyi I have some more upcoming changes that remove this completely.

**Review by bkueng — COMMENTED** [review]
Thanks @mcsauder, looks good.
I'm ok with bringing this in, but I do have some more upcoming changes that conflict quite a bit.

**Comment by mcsauder** [comment]
Closing as superseded by  #20173

**Pull request closed without merging**